### ISSUE TITLE
[bug]: Have problems for a self hosted installation

### ISSUE THREAD

**Issue by giosueDelgado** [hoppscotch/hoppscotch#3342]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I create a docker compose where I want to run my seld hosted installation.

Here the docker-compose:
```

hoppscotch:
    image: hoppscotch/hoppscotch:2023.8.0
    environment:
      - DATABASE_URL=postgresql://${LINKDB}
      - JWT_SECRET=
      - TOKEN_SALT_COMPLEXITY=10
      - MAGIC_LINK_TOKEN_VALIDITY=3
      - REFRESH_TOKEN_VALIDITY=604800000
      - ACCESS_TOKEN_VALIDITY=86400000
      - SESSION_SECRET=
      - REDIRECT_URL=https://test.platform.relybytes.com/gui-api
      - WHITELISTED_ORIGINS=https://test.platform.relybytes.com,https://*.platform.relybytes.com,https://platform.relybytes.com
      - VITE_ALLOWED_AUTH_PROVIDERS=EMAIL
      - MAILER_SMTP_URL=smtps://${credentialemail}
      - MAILER_ADDRESS_FROM='"${sendeemail}'
      - RATE_LIMIT_TTL=60
      - RATE_LIMIT_MAX=100
      - VITE_BASE_URL=https://test.platform.relybytes.com/gui-api
      - VITE_SHORTCODE_BASE_URL=https://test.platform.relybytes.com/gui-api
      - VITE_ADMIN_URL=https://test.platform.relybytes.com/gui-api-admin
      - VITE_BACKEND_GQL_URL=https://test.platform.relybytes.com/gui-api-backend/graphql
      - VITE_BACKEND_WS_URL=wss://test.platform.relybytes.com/gui-api-backend/graphql
      - VITE_BACKEND_API_URL=https://test.platform.relybytes.com/gui-api-backend/v1
```

And I have a nginx that redirect the traffic based on path location to the port of this container "/gui-api" to 3000, "/gui-api-admin" to 3100 and "/gui-api-backend" to 3170

But I have some issue on base path for the GUI application (to make work I have to replace the link in the html and in the css to add the path).

Is it possible to have a full guide documentation to have this solution in the self hosted with a working example that can work also on path managed and not on port managed solution?
Because following the documentation not work.
Thanks

### Steps to reproduce

See the description

### Environment

Production

### Version

Self-hosted

**Comment by Zer0S2m** [comment]
Hello @giosueDelgado!
I have the same problem. The simple solution that I was able to do was to create two subdomains, the first for the application itself, the second for the admin area.

**Comment by giosueDelgado** [comment]
Hi @Zer0S2m thank you for your suggestion, yes I think it is a good workaround, I think we can use this issue also to give a configuration option for those who cannot manage many domains but only one and work on subpath..

**Comment by jamesgeorge007** [comment]
Hi, we're adding the ability to access the services under subpaths. The changes have been merged in #3449 and will be going with the `2023.12.0` release scheduled for December.

**Comment by giosueDelgado** [comment]
Cool! Thanks! @jamesgeorge007
